From 78c89ce6d772d1e591672f72a48b2e8b88aab016 Mon Sep 17 00:00:00 2001
From: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
Date: Thu, 15 Jun 2023 18:27:12 -0300
Subject: [PATCH] csm-replication: fix helm lint error

This patch consists of removing the dashes, because it didn't pass
by currently helm lint version in the system.

Signed-off-by: Hediberto Cavalcante da Silva <hediberto.cavalcantedasilva@windriver.com>
Signed-off-by: Gustavo Ornaghi Antunes <gustavo.ornaghiantunes@windriver.com>
---
 helm/csm-replication/templates/controller.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/helm/csm-replication/templates/controller.yaml b/helm/csm-replication/templates/controller.yaml
index bd33bd0..9fca1f5 100644
--- a/helm/csm-replication/templates/controller.yaml
+++ b/helm/csm-replication/templates/controller.yaml
@@ -213,7 +213,7 @@ subjects:
   name: dell-replication-controller-sa
   namespace: {{ .Release.Namespace }}
 ---
-{{- $secret := (lookup "v1" "ConfigMap" "dell-replication-controller" "dell-replication-controller-config").data -}}
+{{ $secret := (lookup "v1" "ConfigMap" "dell-replication-controller" "dell-replication-controller-config").data }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
-- 
2.25.1
